configs/tasks/rearrange_easy.local.rgbd.yaml

```diff
@@ -143,6 +113,7 @@ SIMULATOR:
         HEIGHT: 1.5
         RADIUS: 0.3
         SENSORS:
+        - HEAD_RGB_SENSOR
```

Comment:
Why are we adding the RGB sensor here while removing it in the hab2_monolithic dockerfile?